test_app/tests/rbac/remote/test_service_api.py

```diff
@@ -289,6 +290,32 @@ def test_service_api_permissions(reverse_name, normal_case, unauth_case, admin_a
     assert unauth_response.status_code == unauth_case, unauth_response.data
 
 
+@pytest.mark.django_db
+def test_role_types_and_permissions_payload_shape(user_api_client):
+    """Minimal payload-shape checks for role types and permissions when accessed by normal user."""
+    # role types
+    url_ct = get_relative_url('dabcontenttype-list')
+    resp_ct = user_api_client.get(url_ct)
+    assert resp_ct.status_code == 200, resp_ct.data
```

Comment:
This test doesn't seem to be any different (doing `user_api_client.get`) from the `('dabcontenttype-list', 200, 401),` params (see the 200 for normal user) of `test_service_api_permissions` test above it. So I have some concern that this isn't really verifying the fix. Although maybe this did fix the issue, I can't say either way.